pubspec.yaml

```
@@ -39,7 +39,19 @@ melos:
 
     integration_test:
       run: melos exec --dir-exists=integration_test --flutter -- "flutter test --dart-define-from-file=../../../config.json integration_test/suite_test.dart"
-      description: Run `integration tests` in all packages that have the folder `integration_test``
+      description: Run `integration tests` in all packages that have the folder `integration_test`
+
+    start_backbone:
+      run: docker compose -f .dev/compose.backbone.yml up -d
+      description: Start Backbone in a Docker container
+
+    stop_backbone:
+      run: docker compose -f .dev/compose.backbone.yml down -v
+      description: Stop Backbone Docker container
+
+    integration_test_local_backbone:
+      run: melos exec --dir-exists=integration_test --flutter --depends-on integration_test -- "flutter test --dart-define=app_baseUrl=http://localhost:8090 --dart-define=app_clientId=test --dart-define=app_clientSecret=test integration_test/suite_test.dart"
+      description: Run `integration tests` in all packages that have the folder `integration_test`
```

To distinguish the description of script `integration_test_local_backbone` from the description of script `integration_test`.

```suggestion
      description: Run `integration tests` with local Backbone in all packages that have the folder `integration_test`
```